.github/workflows/ci.yaml

```diff
@@ -280,7 +280,7 @@ jobs:
         uses: benjefferies/branch-protection-bot@1.0.7
         with:
           access_token: ${{ secrets.API_TOKEN_GITHUB }}
-          owner: biomage-ltd
+          owner: hms-dbmi-cellenics
```

Comment:
Is the repository owner `biomage-ltd` the organization? Or is it an account with the same name?

Reply:
@ivababukova